### PR TITLE
Add example webserver configuration

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -216,6 +216,7 @@
     * [Filesystem](guides/hosting/infrastructure/filesystem.md)
     * [Message Queue](guides/hosting/infrastructure/message-queue.md)
     * [Reverse Http Cache](guides/hosting/infrastructure/reverse-http-cache.md)
+    * [Elasticsearch](guides/hosting/infrastructure/elasticsearch.md)
```
Comment (elkmod) [inline SUMMARY.md:219]: Why adding Elastichsearch here?

File: resources/references/config-reference/server/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Config Reference
```
Comment (elkmod) [inline resources/references/config-reference/server/README.md:1]: Server Config Reference

File: resources/references/config-reference/server/apache.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Apache config reference
```
Comment (elkmod) [inline resources/references/config-reference/server/apache.md:1]: Apache Config Reference

File: resources/references/config-reference/server/nginx.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# Nginx config reference
```
Comment (elkmod) [inline resources/references/config-reference/server/nginx.md:1]: Nginx Config Reference

**Review by elkmod — CHANGES_REQUESTED** [review]

File: SUMMARY.md (hunk 1 above)
Comment (shyim) [reply SUMMARY.md:219]: was missing

**Pull request merged**